drivers/vfio/vfio.c

```diff
@@ -827,31 +770,29 @@ int vfio_add_group_dev(struct device *dev,
 		iommu_group_put(iommu_group);
```

Comment:
why need to "put" it here? shouldn't `vfio_group_get_from_iommu` handle that correctly?

Reply:
ok, looks like every `iommu_group_get` leads to ref counter increment.
so you keep it if no vfio group exists and dec it otherwise